From e162696827d97449e6395fc017fe6865aa6f1ad1 Mon Sep 17 00:00:00 2001
From: Frantisek Sumsal <frantisek@sumsal.cz>
Date: Tue, 21 Jun 2022 10:01:30 +0200
Subject: [PATCH] test: always force a new image for dfuzzer

Otherwise we might end up with an image containing broken service
symlinks and other things, which break certain parts of the test.

(cherry picked from commit 5dffa6b032168305213e4fb0d72fb02363acfd65)

Related: #2087652
---
 test/TEST-21-DFUZZER/test.sh | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/test/TEST-21-DFUZZER/test.sh b/test/TEST-21-DFUZZER/test.sh
index 42e37c8a9c..7669e4e0ad 100755
--- a/test/TEST-21-DFUZZER/test.sh
+++ b/test/TEST-21-DFUZZER/test.sh
@@ -6,6 +6,8 @@ TEST_DESCRIPTION="Fuzz our D-Bus interfaces with dfuzzer"
 TEST_NO_NSPAWN=1
 TEST_SUPPORTING_SERVICES_SHOULD_BE_MASKED=0
 QEMU_TIMEOUT="${QEMU_TIMEOUT:-1800}"
+IMAGE_NAME=dfuzzer
+TEST_FORCE_NEWIMAGE=1
 
 # shellcheck source=test/test-functions
 . "${TEST_BASE_DIR:?}/test-functions"
